examples/cpuexporter.lua

```diff
@@ -0,0 +1,173 @@
+--
+-- SPDX-FileCopyrightText: (c) 2025 Enderson Maia <endersonmaia@gmail.com>
+-- SPDX-License-Identifier: MIT OR GPL-2.0-only
+--
+
+local lunatik = require("lunatik")
+local thread  = require("thread")
+local socket  = require("socket")
+local unix    = require("socket.unix")
+local linux   = require("linux")
+local cpu     = require("cpu")
+
+local shouldstop = thread.shouldstop
+local sock = socket.sock
+
+local server = unix.bind("/tmp/cpuexporter.sock", "STREAM")
+server:listen()
+
+local last_stats = {}
+
+-- Scale integer to decimal with high precision
+-- Converts a ratio (value/total) to percentage format with 16 decimal places
+local function to_percent_float(value, total)
```

Comment:
I think this name is a bit misnomer.. I would call this something like `format_percentage` or something.. you're not converting to float, but formatting it..